webgoat-lessons/xxe/src/main/resources/lessonPlans/en/XXE_intro.adoc

```diff
@@ -1,38 +1,24 @@
 === What is a XML entity?
 
-An XML Entity allows tags to be defined that will be replaced by content when the XML Document is parsed.
-In general there are three types of entities:
+An XML Entity allows tags to be defined that will be replaced by content when the XML Document is parsed. In general there are three types of entities:
 
 * internal entities
 * external entities
 * parameter entities.
 
 An entity must be created in the Document Type Definition (DTD), let's start with an example:
 
-[source]
-----
-<?xml version="1.0" standalone="yes" ?>
-<!DOCTYPE author [
-  <!ELEMENT author (#PCDATA)>
-  <!ENTITY js "Jo Smith">
-]>
-<author>&js;</author>
-----
+[role="lesson-image"]
+image::images/xxe-parser.png[XML parser]
+
+As you can see once the XML document is processed by the parser, it will replace the defined entity `js` with the defined constant "Jo Smith". As you can see this has many advantages as you can change `js` in one place to for example "John Smith".
```

Comment:
It might be nice to mention or remind people that the parsing of XML in Java applications is used to transform content from text files into Java objects. Maybe add a little java object to the output of the XML parser. Something like: 
<img width="727" alt="Schermafbeelding 2020-05-18 om 12 01 59" src="https://user-images.githubusercontent.com/7931455/82201414-9eb60480-9900-11ea-9c94-5b300af33a58.png">

Reply:
added in the same style as the previous picture